src/main/java/com/amam/wizardschool/controller/FacultyController.java

```diff
@@ -19,30 +21,35 @@ public FacultyController(FacultyService facultyService) {
     }
 
     @GetMapping("/all")
```

Comment:
```suggestion
    @GetMapping
```
никаких слов в пути быть не должно, это нарушает REST